lv_conf_template.h

```diff
@@ -140,6 +140,10 @@
 
 /* Enable VG-Lite assert. */
 #define LV_VG_LITE_USE_ASSERT 0
+
+/* Simulate VG-Lite hardware using ThorVG */
+#define LV_USE_VG_LITE_TVG  0
```

Comment:
```suggestion
#define LV_USE_VG_LITE_THORVG  0
```
as we are using the thorvg name on other places too.